nav2_msgs/msg/WaypointStatus.msg

```diff
@@ -0,0 +1,10 @@
+# Waypoint state for robot in NavigateThroughPoses or WaypointFollower
+uint8 PENDING=0 # Waypoint is not processed or processing
+uint8 COMPLETED=1 # Waypoint is completed
+uint8 SKIPPED=2 # Waypoint is skipped
+uint8 MISSED=3 # Waypoint is missed
```

Comment:
I think MISSED is redundant with SKIPPED, no? 

I think we could keep MISSED and remove SKIPPED. What do you think?